examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -1646,6 +1646,42 @@ otThreadSetMasterKey(
 }
 
 OTAPI 
+const uint8_t *
+OTCALL
+otThreadGetPSKc(
+    _In_ otInstance *aInstance 
+    )
+{
+    if (aInstance == nullptr) return nullptr;
+
+    uint8_t *Result = (uint8_t*)malloc(sizeof(otPSKc));
+    if (Result == nullptr) return nullptr;
+    if (QueryIOCTL(aInstance, IOCTL_OTLWF_OT_PSKC, Result) != ERROR_SUCCESS)
+    {
+        free(Result);
+        return nullptr;
```

Comment:
We don't use the same code style in the Windows code. This new code is following the existing code style.